conf/base.config

```
@@ -11,24 +11,54 @@
 
 process {
 
-  // TODO nf-core: Check the defaults for all processes
+ // Process-specific containers
+withLabel: openms      { container = "metaboigniter/container-openms:v2.4.0" }
```

Check with @ewels whether its ok to do it like this instead of environment.yml and dockerfile

---

We have already been in contact several times. I think we came to the conclusion that making conda environment is difficult as we have many inconsistent packages. So it's okay like this. The solution would be to update the whole workflow which is on the agenda for the DSL 2

---

ok then fine with me as well

---

I think it would be good to have these under the `nfcore` DockerHub account if possible. Mostly because we have `nfcore` registered as an open-source community there so the account is not affected by pull limits and auto-archive / deletion policies. Without this I'd worry that the images might get automatically removed in a year or two. For example:

```suggestion
withLabel: openms      { container = "nfcore/metaboigniter-openms:v2.4.0" }
```

It would also be good to have the `Dockerfile` scripts in the pipeline for clarity (eg. in a subfolder).

Happy to help with both of these points.

---

Fixed